From f97a36967e34b750f33b59802be4dfaf174efaee Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 10 Oct 2017 14:18:43 -0700
Subject: [PATCH] TOOLS-1883 sdcadm shouldn't hardcode path to psql Reviewed
 by: Cody Peter Mello <cody.mello@joyent.com> Approved by: Cody Peter Mello
 <cody.mello@joyent.com>

---
 lib/common.js | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/lib/common.js b/lib/common.js
index 71180e6..cce8d49 100644
--- a/lib/common.js
+++ b/lib/common.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2017 Joyent, Inc.
+ * Copyright (c) 2017, Joyent, Inc.
  */
 
 var assert = require('assert-plus');
@@ -996,7 +996,10 @@ function waitForPostgresUp(opts, cb) {
             '-n',
             opts.server,
             /* JSSTYLED */
-            format('/usr/sbin/zlogin %s "/opt/local/bin/psql -U postgres -t -A -c \'SELECT NOW() AS when;\'"', opts.vm)
+            format('/usr/sbin/zlogin %s '
+                + '\'PATH=/opt/postgresql/current/bin:/opt/local/bin:$PATH '
+                + 'psql -U postgres -t -A -c "SELECT NOW() AS when;"\'',
+                opts.vm)
         ];
 
         execFilePlus({
-- 
2.21.0

